src/AutomaticMechanismAnalysis.jl

```diff
@@ -280,7 +280,7 @@ function analyzespc(sim,spcname,t;N=10,tol=1e-3,branchthreshold=0.9,
     for rxnind in sensinds
         branches,rps = getbranchpathinfo(sim,spcind,rxnind,ropp,ropl,rts;steptol=steptol,branchtol=branchtol)
         radprodlossfract = getradprodlossfract(sim,rxnind,rts)
-        if eliminate
+        if eliminate && !isempty(rps)
```

Comment:
What if we want to eliminate unimportant branches even if we don't have any reaction paths?

Reply:
You're right, I will remove this